api/nodes/serializers.py

```diff
@@ -63,10 +40,12 @@ class NodeSerializer(JSONAPISerializer):
                                                             'public and private nodes. Administrators on a parent '
                                                             'node have implicit read permissions for all child nodes',
                               )
-    # TODO: finish me
 
-    class Meta:
-        type_ = 'nodes'
+    registration = ser.BooleanField(source='is_registration', read_only=True)
```

Comment:
Of course, the answer is, "everything". But I'll poke through some APIs I like and see if they do anything somewhat consistently.